test/recipes/90-test_sslapi.t

```diff
@@ -33,14 +34,32 @@ ok(run(test(["sslapitest", srctop_dir("test", "certs"),
              srctop_file("test", "recipes", "90-test_sslapi_data",
                          "passwd.txt"), $tmpfilename, "default",
              srctop_file("test", "default.cnf")])),
-             "running sslapitest");
+             "running sslapitest")
+    if $no_dh;
+
+ok(run(test(["sslapitest", srctop_dir("test", "certs"),
+             srctop_file("test", "recipes", "90-test_sslapi_data",
+                         "passwd.txt"), $tmpfilename, "default",
+             srctop_file("test", "default.cnf"),
+             srctop_file("test", "recipes", "90-test_sslapi_data", "dhparams.pem")])),
+             "running sslapitest")
+    unless $no_dh;
```

Comment:
Instead of two invocations with and without dhparams here I would just handle the no-dh in the sslapitest.c